File: Siscon_API_Backend/Siscon_API_Backend/Models/Apartamento.cs
namespace Siscon_API_Backend.Models
{
    public class Apartamento
    {
        public long Id { get; set; }
        public int? NumeroApto { get; set; }
        public int? Andar { get; set; }
        public long? PredioId { get; set; }

        public Predio Predio { get; set; }
    }
}

File: FRONTEND/SisconFrontEnd/SisconFrontEnd/src/app/models/Apartamento.ts
import { Predio } from '../models/Predio';;

export class Apartamento {
  id: number;
  numeroApto: number | null;
  andar: number | null;
  predioId: number | null;
  predio: Predio
}

export class ApartamentoPayload {
  data: Apartamento[];
}

export class ApartamentoPayloadUniqueResult {
  data: Apartamento;
}
